### DISCUSSION TITLE
How to include a clj file?

### DISCUSSION THREAD

**Discussion by omarpta** [babashka/babashka#875] [Q&A] [answered]
How can I include a clj file inside another folder in babashka?
eg: /path/to/otherfile.clj

Thanks

**Comment by borkdude** [comment, marked as answer]
You can do this in two ways:

- load a file directly using `(load-file "the-file.clj")`
- add `/path/to` to the classpath and then use `(require 'otherfile)` 

You can set the classpath by making a `bb.edn` file that looks like:

```
{:paths ["path/to"]}
``` 

or by using `(babashka.classpath/add-classpath "path/to")`

or by using the `--classpath` CLI argument.

Also see http://book.babashka.org.

**Comment by omarpta** [comment]
Thanks a lot Michiel!
